base/standard/src/main/java/org/openscience/cdk/graph/invariant/Canon.java

```diff
@@ -120,7 +123,7 @@ private Canon(int[][] g, long[] partition, boolean[] hydrogens, boolean symOnly)
     /**
      * Compute the canonical labels for the provided structure. The labelling
      * does not consider isomer information or stereochemistry. This method
-     * allows provision of a custom array of initial invariants.
+     * allows provision of a custom array of initial initial.
```

Comment:
-> "initial seeds"?

Reply:
Stupid IDE likes to change all occurrences by default. Should be initial invariants.